### ISSUE TITLE
Classes not exported warnings

### ISSUE THREAD

**Issue by Ahlisen** [react-native-community/react-native-share#329]
When updating my project to react-native 56.0 and RN-share to 1.1.0 I get warnings about RCT_EXPORT_MODULE not being used in Instagram, GooglePlus, WhatsApp and Generic-Share (See Image). Is this normal with the new RN 56.0?

### Environment
- **React Native version**: 56.0
- **React Native platform + platform version**: iOS 9.0

### react-native-share
**Version**: 1.1.0

![warnings](https://user-images.githubusercontent.com/10942576/43655907-c4fa6db0-9750-11e8-905d-52f5205692bd.jpg)

**Comment by jgcmarins** [comment]
done https://github.com/react-native-community/react-native-share/pull/331

**Comment by lightman73** [comment]
Hello, 

 although I'm not seeing the RNShare warning doesn't appear, the same RCT_EXPORT_MODULE warnings are still there even with v. 1.1.1 (using RN 0.55.4). 

 Any ideas on how to get rid of them ? 

Thanks !

**Comment by Shazam14** [comment]
I am also having the same issue

**Comment by klabusta** [comment]
Yes, same setup: 1.1.1 (using RN 0.55.4). I'm seeing those warnings too. Any way I can disable those?

**Comment by eminsr** [comment]
I have the same warnings.

```
react-native: 0.56.0
iOS: 11.4
react-native-share: 1.1.2
```

If you choose to ignore them you can add the following but I'm not sure you should tho. Also, this may be related to [#14806](https://github.com/facebook/react-native/issues/14806) and [#18201](https://github.com/facebook/react-native/issues/18201)?

```
import { YellowBox } from "react-native";
YellowBox.ignoreWarnings([
  "Class GenericShare",
  "Class GooglePlusShare",
  "Class WhatsAppShare",
  "Class InstagramShare"
]);
```

**Comment by jgcmarins** [comment]
#370 merged, can anyone test master branch to see if it is ok?
Thanks.

**Comment by gtsluk** [comment]
Followed the changeling, 
I add 'RCT_EXPORT_MODULE();' under the @implemengtation.
The warning does not appear in first load.

**Comment by devotebest** [comment]
> I have the same warnings.
> 
> ```
> react-native: 0.56.0 & 0.57.0
> iOS: 11.4 & 12.0
> react-native-share: 1.1.2
> ```
> If you choose to ignore them you can add the following but I'm not sure you should tho. Also, this may be related to [#14806](https://github.com/facebook/react-native/issues/14806) and [#18201](https://github.com/facebook/react-native/issues/18201)?
> 
> ```
> import { YellowBox } from "react-native";
> YellowBox.ignoreWarnings([
>   "Class GenericShare",
>   "Class GooglePlusShare",
>   "Class WhatsAppShare",
>   "Class InstagramShare"
> ]);
> ```



> I have the same warnings.
> 
> ```
> react-native: 0.56.0 & 0.57.0
> iOS: 11.4 & 12.0
> react-native-share: 1.1.2
> ```
> If you choose to ignore them you can add the following but I'm not sure you should tho. Also, this may be related to [#14806](https://github.com/facebook/react-native/issues/14806) and [#18201](https://github.com/facebook/react-native/issues/18201)?
> 
> ```
> import { YellowBox } from "react-native";
> YellowBox.ignoreWarnings([
>   "Class GenericShare",
>   "Class GooglePlusShare",
>   "Class WhatsAppShare",
>   "Class InstagramShare"
> ]);
> ```

which file for this?

**Comment by jgcmarins** [comment]
> which file for this?

Root? Like App.js.
